components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -845,7 +903,8 @@ private List<EmailTemplate> getAllTemplatesOfTemplateTypeFromRegistry(String tem
         }
         for (String template : templateType.getChildren()) {
             Resource templateResource = resourceMgtService.getIdentityResource(template, tenantDomain);
-            if (templateResource != null) {
+            if (templateResource != null && (templateTypeRegistryPath.contains(APP_TEMPLATE_PATH)
```

Comment:
Explain this in a comment.

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/223/commits/43afd46f6a19a8fc329ccee7166d75003c866306